src/components/OntologyBrowser/SelectionTree.tsx

```diff
@@ -146,25 +145,52 @@ export class SelectionTree extends Component<SelectionTreeProps, SelectionTreeSt
     }
   };
 
-  onExpand = (expand: (string | number)[], info: Object) => {
+  onExpand = (expand: (string | number)[]) => {
     this.setState({
       expandedKeys: expand.map((v) => v.toString()),
     });
   };
 
+  getCheckedKeys = (
+    key: string | number,
+    dataSource = this.props.dataSource,
+    accu: { check: string[]; halfcheck: string[] } = {
```

Comment:
imo, i'll keep a name like checkKeys or something like that, It's more a javascripty than scalaist (if I can say that)